modules/virt-setting-openshiftv-lm-feature-gates.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-enabling-cclm-for-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-setting-openshiftv-lm-feature-gates_{context}"]
+= Setting a live migration feature gate for each cluster in {VirtProductName}
+
+To enable cross-cluster live migration, you must set a feature gate for each of the two clusters in {VirtProductName}.
```

Comment:
Just noticed this is missing the `[role="_abstract"]` - please check these for all the modules